File: Mobile MediaPlayer (Xamarin Android)/MediaPlayer/MediaPlayer/Core/CurrentInfo.cs
using System.Collections.Generic;

namespace MediaPlayer.Core
{
    class CurrentInfo
    {
        public enum States { SongPlaying, PlaylistCreation };
        public enum PlayingModes { NormalMode, ShuffleMode };

        public States State;
        public Playlist Playlist;
        public int SongIndex;
        public PlayingModes PlayingMode;
        public List<bool> PlayedSongs;
        public int LastSong;
    }
}

File: Mobile MediaPlayer (Java Android)/app/src/main/java/com/example/mediaplayer/Core/CurrentInfo.java
package com.example.mediaplayer.Core;

import java.util.List;

public class CurrentInfo {
    public enum States { SongPlaying, PlaylistCreation }
    public enum PlayingModes { NormalMode, ShuffleMode }

    public States State;
    public Playlist Playlist;
    public int SongIndex;
    public PlayingModes PlayingMode;
    public List<Boolean> PlayedSongs;
    public int LastSong;
}
